config/samples/logstash/logstash_es.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: elasticsearch-sample
+spec:
+  version: 8.7.0
+  nodeSets:
+    - name: default
+      count: 2
+      config:
+        node.store.allow_mmap: false
+---
+apiVersion: logstash.k8s.elastic.co/v1alpha1
+kind: Logstash
+metadata:
+  name: logstash-sample
+spec:
+  count: 1
+  version: 8.7.0
+  elasticsearchRefs:
+    - name: elasticsearch-sample
+  pipelines:
+    - pipeline.id: main
+      config.string: |
+        input { exec { command => 'uptime' interval => 10 } } 
+        output { 
+          elasticsearch {
+            hosts => [ "${DEFAULT_ELASTICSEARCH_SAMPLE_ES_HOSTS}" ]
```

Comment:
nit: let's add a comment to explain that this example is expected to be deployed in the `default` namespace. The env. variables must be updated if it is not the case.